compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/infer-effect-dependencies/bailout-retry/error.use-no-memo.expect.md

```diff
@@ -0,0 +1,27 @@
+
+## Input
+
+```javascript
+// @inferEffectDependencies @panicThreshold(none)
+import {useEffect} from 'react';
+
+function Component({propVal}) {
+  'use no memo';
```

Comment:
Does this mean `'use no memo'` should not disable autodeps/fire? (that sounds right to me, just checking if that's your intent)

Reply:
Yep that's right -- ideally we'd still run the compiler on `use no memo` (just with memoization disabled), but that's still a todo